api/src/main/java/marquez/service/LineageService.java

```diff
@@ -85,19 +99,23 @@ public Lineage lineage(NodeId nodeId, int depth, boolean withRunFacets) {
     if (nodeId.isDatasetType()
         && datasets.stream().noneMatch(n -> n.getId().equals(nodeId.asDatasetId()))) {
       log.warn(
-          "Found jobs {} which no longer share lineage with dataset {} - discarding",
-          jobData.stream().map(JobData::getId).toList());
-      DatasetId datasetId = nodeId.asDatasetId();
-      DatasetData datasetData =
-          getDatasetData(datasetId.getNamespace().getValue(), datasetId.getName().getValue());
-      return new Lineage(
-          ImmutableSortedSet.of(
-              Node.dataset().data(datasetData).id(NodeId.of(datasetData.getId())).build()));
+          "Found jobs '{}' which no longer share lineage with dataset '{}' - discarding",
```

Comment:
The jobs will be a list, surrounded by array brackets - I don't see a point in quoting them.